mdoc/src/main/scala/mdoc/Exit.scala

```diff
@@ -0,0 +1,18 @@
+package mdoc
+
+import mdoc.internal.cli.{Exit => CliExit}
+
+final class Exit private[mdoc] (val exit: CliExit) {
+  def merge(other: Exit): Exit = Exit(exit.merge(other.exit))
+  def isSuccess: Boolean = exit == CliExit.success
+  def isError: Boolean = exit == CliExit.error
+}
+
+object Exit {
+  def apply(exit: CliExit): Exit = {
```

Comment:
Had to make this package private because it is used [here](https://github.com/hmf/mdoc/blob/post-life-cycle/mdoc/src/main/scala/mdoc/internal/cli/MainOps.scala#L237) in `MainOps`.